.github/workflows/ci.yml

```diff
@@ -21,15 +21,26 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       # Setup go and stringer
-      - uses: actions/setup-go@v3
-      - run: go get -v golang.org/x/tools/cmd/stringer
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v3.3.1
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Setup stringer
+        run: go install golang.org/x/tools/cmd/stringer@latest
 
       # Setup python and reuse
       - uses: actions/setup-python@v3
       - uses: BSFishy/pip-action@v1
         with:
           packages: reuse
 
+      - name: Check for go vulnerabilities
+        run: |
+          go install golang.org/x/vuln/cmd/govulncheck@latest
+          govulncheck -v -test ./...
+
```

Comment:
Please create an extra GHA `job` for it, since it does not belong to `generate-required`